MovieChecker/MovieChecker/BoxOfficeTableCell.m

```diff
@@ -0,0 +1,37 @@
+//
+//  BoxOfficeCell.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "BoxOfficeTableCell.h"
+
+#import <SDWebImage/UIImageView+WebCache.h>
+
+@interface BoxOfficeTableCell ()
+
+@property (nonatomic, weak) IBOutlet UILabel *nameLabel;
+@property (nonatomic, weak) IBOutlet UIImageView *thumbnailImageView;
+
+@end
+
+@implementation BoxOfficeTableCell
+
+- (void)setThumbnailImageFromURL:(NSURL *)url
+{
+    [self.thumbnailImageView sd_setImageWithURL:url completed:^(UIImage *image,
+                                                                NSError *error,
+                                                                SDImageCacheType cacheType,
+                                                                NSURL *imageURL) {
+        self.thumbnailImageView.image = image;
```

Comment:
Probably safer as a weak reference to `self`...  and, actually, as much overlap as there is in your two cell classes, it would probably make sense to have a common base cell class to take care of the common code.